### ISSUE TITLE
twisted.web2.xmlrpc.XMLRPC._cbRender uses a bare except and doesn't log the failure.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2214]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)| @dreid reported|
|-|-|
|Trac ID|trac#2214|
|Type|defect|
|Created|2006-11-03 05:00:19Z|

twisted.web2.xmlrpc.XMLRPC._cbRender should log the exception even if it doesn't send it over the wire.   Otherwise debugging is a major PITA.

<details><summary>Searchable metadata</summary>

```
trac-id__2214 2214
type__defect defect
reporter__dreid dreid
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__web2 web2
keywords__ 
time__1162530019000000 1162530019000000
changetime__1297620781000000 1297620781000000
version__None None
owner__ 
cc__oubiwann
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)<a name="note_1"></a>|@dreid commented|
|-|-|

The error path for this also isn't tested.

**Comment by twisted-trac** [comment]
|[<img alt="oubiwann's avatar" src="https://avatars.githubusercontent.com/u/894251?s=50" width="50" height="50">](https://github.com/oubiwann)|@oubiwann set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

Not going to fix this (see [#4821](https://github.com/twisted/twisted/issues/4821)).  The Twisted Web XMLRPC support does log errors in the equivalent case, so no need to retarget this at the web component.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
